core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.ts

```diff
@@ -16,28 +34,88 @@ import { DotIconModule } from '@dotcms/ui';
         DotExperimentsConfigurationItemsCountComponent,
         DotMessagePipeModule,
         DotIconModule,
+        DotDynamicDirective,
+        UiDotIconButtonModule,
         //PrimeNg
         CardModule,
-        ButtonModule
+        ButtonModule,
+        UiDotIconButtonTooltipModule
     ],
     templateUrl: './dot-experiments-configuration-variants.component.html',
     styleUrls: ['./dot-experiments-configuration-variants.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class DotExperimentsConfigurationVariantsComponent {
-    maxLength = MAX_VARIANTS_ALLOWED;
-    @Input()
-    trafficProportion: TrafficProportion;
+export class DotExperimentsConfigurationVariantsComponent implements OnDestroy {
+    @ViewChild(DotDynamicDirective, { static: true })
+    dotDynamicHost!: DotDynamicDirective;
 
+    vm$ = this.dotExperimentsConfigurationStore.vmVariants$;
+
+    maxVariantsAllowed = MAX_VARIANTS_ALLOWED;
+    defaultVariantId = DEFAULT_VARIANT_ID;
+
+    private destroy$: Subject<boolean> = new Subject<boolean>();
+
+    constructor(
+        private readonly dotExperimentsConfigurationStore: DotExperimentsConfigurationStore,
```

Comment:
You should not be using this sore in two different components, also the store shuld be instantiated here in the component that uses it.